independent-publisher-connectors/Google Photos/apiDefinition.swagger.json

```diff
@@ -0,0 +1,161 @@
+        "operationId": "ListAlbums",
+        "parameters": [],
+        "responses": {
+          "default": {
+            "description": "200",
+            "schema": {}
+          }
+        }
+      },
+      "post": {
+        "summary": "Create New Album",
+        "description": "Creates an album in a user's Google Photos library.\n\n### HTTP request\n\n`POST https://photoslibrary.googleapis.com/v1/albums`\n\nThe URL uses [gRPC Transcoding](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) syntax.\n\n### Request body\n\nThe request body contains data with the following structure:\n\nJSON representation\n\n```json\n{\n  \"album\": {\n    object (Album)\n  }\n}\n```\n\n|Fields|||\n|----|----|----|\n|`album`|`object` ([Album](https://developers.google.com/photos/library/reference/rest/v1/albums#Album))|The album to be created.|\n\n### Response body\n\nIf successful, the response body contains a newly created instance of [Album](https://developers.google.com/photos/library/reference/rest/v1/albums#Album).\n\n### Authorization Scopes\n\nRequires one of the following OAuth scopes:\n\n*   `https://www.googleapis.com/auth/photoslibrary`\n*   `https://www.googleapis.com/auth/photoslibrary.appendonly`\n*   `https://www.googleapis.com/auth/photoslibrary.sharing`",
+        "operationId": "CreateAlbum",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "string"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/v1/albums/%7B%7BalbumId%7D%7D": {},

```

Comment:
@juliamuiruri4 can you please explain why you have an empty paths? May I ask you please to remove all empty paths from your swagger file.